README.rst

```diff
@@ -1,6 +1,15 @@
 requests-ntlm
 =============
 
+.. image:: https://travis-ci.org/jborean93/requests-ntlm.svg?branch=master
+    :target: https://travis-ci.org/jborean93/requests-ntlm
+
+.. image:: https://ci.appveyor.com/api/projects/status/nj172p4nkyalwh6r/branch/master?svg=true
+    :target: https://ci.appveyor.com/project/jborean93/requests-ntlm
+
+.. image:: https://coveralls.io/repos/github/jborean93/requests-ntlm/badge.svg?branch=master
+    :target: https://coveralls.io/github/jborean93/requests-ntlm?branch=master
```

Comment:
Yep, I can easily change the travis-ci and coveralls but would need the token value for appveyor that is set up for the organisation (requests). Not sure who would have permission to give access for that.